lib/persister/game_data.rb

```diff
@@ -0,0 +1,108 @@
+require_relative '../game_mode/lite_3';
+
+class GameData
+    def initialize(game)
+        @game = game
+    end
+
+    def types
+        [
+            board_presenter,
+            status,
+            game_mode,
+            board_positions,
+            board_size,
+            current_player_type,
+            current_player_token,
+            player_1_moves,
+            player_2_moves
+        ]
+    end
+
+    private
+    attr_accessor :game
+
+    def board_presenter
+        {
+            :name => :board_presenter,
+            :data => game.board_presenter.presenter.class.name
+        }
+    end
+
+    def status
+        {
+            :name => :status,
```

Comment:
`name: :status,`

Reply:
The different ruby syntax

Reply:
to clarify, are you saying its more common to write hashes like `name: :status` over `:name => :status`

Reply:
I prefer it, it's a lot cleaner. Rubocop also suggests to use it